examples/object_detection/main.py

```diff
@@ -142,6 +142,11 @@ def criterion_fn(model_outputs, target, criterion):
             return loss_l + loss_c
 
         def autoq_test_fn(model, eval_loader):
+            # Put BN layers into evaludtion mode
```

Comment:
why is it applicable for object detection only?
does it make sense to do it for all models inside autoq before calling autoq_test_fn?

Reply:
Will move eval mode switch for BNs inside `test_net` in this sample. For other samples, we already have `model.eval()` inside validation methods. I guess it's the responsibility of the user/training sample to do correct validation of the model.